core/pkg/monitor/apple/winibuild.py

```diff
@@ -0,0 +1,47 @@
+"""Builds the AppleStats Swift binary for monitoring system metrics."""
```

Comment:
do you think we could have an option to skip building the applestats (if it is already available). Just thinking for dev it takes a while and it will be quite annoying to wait a few minutes, when you actually just want to build the core logic?

Reply:
The only thing is that our mac tests will be a bit slower (we don't cache anything), but maybe we can live with this.

Reply:
We should definitely add caches to CircleCI later, after cleaning it up. On another note, I do think it might be worth including AppleStats in system tests.